flavors/template-Exasol-all-java-17/flavor_base/language_definition

```diff
@@ -0,0 +1 @@
+JAVA=localzmq+protobuf:///{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}?lang=java#buckets/{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}/exaudf/exaudfclient_py3
```

Comment:
```suggestion
JAVA=localzmq+protobuf:///{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}?lang=java#buckets/{{ bucketfs_name }}/{{ bucket_name }}/{{ path_in_bucket }}{{ release_name }}/exaudf/exaudfclient
```

❓

Reply:
This is an artifact of us having in the past python3 and python2 in the same flavor and changing it would have a high effort, why we didn't do it